NETPROGRAM/ArrayExercise/Exercise08.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.ArrayExercise
+{
+    public class Exercise08
+    {
+
+        public int CountNumbers(int[] array)
+        {
+
+            return (from i in array
```

Comment:
Lúc nào cái query đơn giản thì nên dùng hàm
```suggestion
            return array.Count(x => x % 3 == 0 && x % 5 != 0);
```

Reply:
done